bin/test-workflow/8_app_verify_authentication.sh

```diff
@@ -104,6 +104,7 @@ else
   init_url_with_host_outside_apps="test-app-with-host-outside-apps-branch-summon-init.$TEST_APP_NAMESPACE_NAME.svc.cluster.local:8080"
   sidecar_url="test-app-summon-sidecar.$TEST_APP_NAMESPACE_NAME.svc.cluster.local:8080"
   secretless_url="test-app-secretless-broker.$TEST_APP_NAMESPACE_NAME.svc.cluster.local:8080"
+  secrets_provider_init_url="test-app-secrets-provider-init.$TEST_APP_NAMESPACE_NAME.svc.cluster.local:8080"
```

Comment:
We'll need a `secrets_provider_init_url` definition for `"$PLATFORM" == openshift" case above (Line 99).
We shouldn't need the `init_url` (Line 97) since that's not in our test matrix, and it's probably going to get deprecated real soon. So feel free to use port `8082` for `secrets_provider_init_url`.

Reply:
Sounds good.
Does this deprecation also apply to `init_url_with_host_outside_apps`?

Reply:
Interesting question. I thought about bringing this up earlier, but I think this merits a discussion on whether this can be deleted or not. My thinking is that we should consider deleting this `init_url_with_host_outside_apps` test case. This test case is a bit of a corner case that's more about testing functionality of Conjur and Conjur security policy. All of the other application test cases use an `apps` policy group... and they require that the application Pod matches a specified Namespace, Deployment, ServiceAccount, and authn container name. For the `init_url_with_host_outside_apps`, the authn-k8s container uses a host ID that isn't part of the `apps` group, and it isn't as particular about checking Deployment and ServiceAccount.

For the other `secrets_provider_init_url` application, I looked back at the discussion for using `authn-k8s` as an init container. Apparently the consensus is not to deprecate, but to advise against using authn-k8s as an in it container  in production (i.e. advise using authn-k8s as a sidecar instead in production).

Reply:
Per our discussion - authn-k8s init container is not deprecated, but advised against in production - `init` URLs are maintained.